templates/zerver/api/incoming-webhooks-walkthrough.md

```diff
@@ -594,3 +594,8 @@ raise UnexpectedWebhookEventType(webhook_name, event_type)
 
 `webhook_name` is the name of the integration that raises the exception.
 `event_type` is the name of the unexpected webhook event.
+
+### Working with a locally hosted third-party service
+
+Please refer to [this](https://zulip.readthedocs.io/en/latest/development/using.html) ReadTheDocs
```

Comment:
We generally don't refer to ReadTheDocs explicitly in our docs, and there's a typo on the next line.  I think we should just say "If you're developing a webhook integration for a third-party hosted service and using the [Vagrant development environment], you'll want to read our guide for [remote network connections to a Vagrant guest]."

Reply:
Alright, I'll just copy over the suggested string and fill in the links :+1: